packages/auth-remix/src/server.ts

```diff
@@ -0,0 +1,795 @@
+import type { Client } from "edgedb";
+import {
+  Auth,
+  builtinOAuthProviderNames,
+  type BuiltinOAuthProviderNames,
+  type TokenData,
+  type emailPasswordProviderName,
+} from "@edgedb/auth-core";
+
+import {
+  type RemixAuthOptions,
+  RemixClientAuth,
+  RemixAuthSession,
+} from "./client";
+import { Octokit } from "@octokit/core";
+import { redirect, json } from "@remix-run/node";
```

Comment:
```suggestion
import { redirect, json } from "@remix-run/server-runtime";
```
Octokit isn't needed, and I think it's better to use imports from remix's shared server-runtime package, rather than the runtime specific `@remix-run/node` package.